openquake/hazardlib/source/rupture.py

```diff
@@ -231,7 +231,8 @@ class ParametricProbabilisticRupture(BaseProbabilisticRupture):
         If occurrence rate is not positive.
     """
     __slots__ = Rupture.__slots__ + [
-        'occurrence_rate', 'temporal_occurrence_model']
+        'occurrence_rate', 'temporal_occurrence_model',
+        'rupture_slip_direction']
```

Comment:
Why are you adding this parameter to the ParametricProbabilisticRupture instead of the Rupture class?  https://github.com/gem/oq-hazardlib/blob/master/openquake/hazardlib/source/rupture.py#L37